build.gradle.kts

```diff
@@ -41,6 +41,8 @@ dependencies {
 
     implementation("com.github.samtools:htsjdk:4.0.1")
 
+    implementation("it.unimi.dsi:fastutil:8.5.12")
```

Comment:
You may want to sync with main as I recently merged a change to build.gradle.kts